src/System.Management.Automation/namespaces/FileSystemContentStream.cs

```diff
@@ -1475,7 +1445,7 @@ private int RefillByteBuff()
             else if (_currentEncoding is UnicodeEncoding ||
                 _currentEncoding is UTF32Encoding ||
                 _currentEncoding is ASCIIEncoding ||
-                IsSingleByteCharacterSet())
+                _currentEncoding.IsSingleByte)
```

Comment:
Should we exactly follow previous logic and have:
```suggestion
                (_currentEncoding.IsSingleByte && Platform.IsWindows))
```
Is there else in the class that blocks us supporting SBCS encodings on Unix?

Reply:
This code is for reading content from a file it seems so I think the comment that Linux only supports UTF8 is false since a file can use a different encoding.  I don't think there's a need to check if Windows